### ISSUE TITLE
Get-Verb v6 help topic needs updating to reflect new output type, System.Management.Automation.VerbInfo

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell-Docs#1481]
Up to v5.1, `Get-Verb` output modified `Microsoft.PowerShell.Commands.MemberDefinition` instances, as still documented.

In v6, a new dedicated type is output: `System.Management.Automation.VerbInfo`.

Version(s) of document impacted
------------------------------
- [x ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

**Comment by tylerl0706** [comment]
[This seems to be fixed](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-verb?view=powershell-6).

However, the problem here is that `Get-Verb` has moved from [`Microsoft.PowerShell.Core`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/functions/get-verb?view=powershell-5.1) to [`Microsoft.PowerShell.Utility`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-verb?view=powershell-6).

@sdwheeler @joeyaiello do we want to add a 6 doc to `Get-Verb` in the previous location to point to the new location? I heard something about redirects in the docathon. Is that something we could use?

**Comment by joeyaiello** [comment]
Nah, just keep what's accurate. Redirects are a hack, we don't want to maintain them long term

**Comment by HemantMahawar** [comment]
This issue was fixed by @matt9ucci with #1872
